.clang-format-ignore

```diff
@@ -1,3 +1,8 @@
+cpp/include/generated
```

Comment:
With the gmake build system, we put generated files in a number of directories, not just cpp/include/generated.

```
./test/IceStorm/stress/generated
./test/IceSSL/configuration/generated
./test/IceBox/configuration/generated
./test/IceBox/admin/generated
./test/IceBridge/simple/generated
./include/generated
./src/IceGrid/generated
./src/Glacier2Lib/generated
./src/IceDiscovery/generated
./src/icegriddb/generated
./src/IceStormLib/generated
./src/Ice/generated
./src/IceStorm/generated
./src/IceLocatorDiscovery/generated
./src/IceGridLib/generated
./src/IceBox/generated
```

and more

Reply:
I'm not sure it's worth trying to skip formatting these. They never get caught up in CI and are not committed.

Reply:
I went with @externls suggestions and removed my entries for the generated Slice code, since they aren't checked in.
If you think they should be added back, I've no strong feelings.